gpt_index/VERSION

```diff
@@ -1 +1 @@
-0.0.10
+0.0.11
```

Comment:
i almost wanna call this `0.1.0` since we're adding functionality and not fixing bugs (yet) here (https://semver.org/)

Reply:
Hmm that's a fair point